mini.go

```diff
@@ -26,6 +26,12 @@ var (
 )
 
 func init() {
+
+	wrapper.AddHookToSyslog("tcp", "localhost:10514", syslog.LOG_EMERG, "Exact Details: ")
```

Comment:
this returns an error if you can't init syslog, not sure how you want to handle that